package.json

```diff
@@ -70,6 +76,11 @@
 		"parserOptions": {
 			"project": "./tsconfig.json"
 		},
+		"ignorePatterns": [
+			"vite.config.js",
+			"build",
+			"dist"
```

Comment:
Ah here's how we prevent linting from going beyond what we want.

As an aside -- I'm not a big fan of using package.json for eslint config (as opposed to an `.eslintrc.json`).  That's out of scope for this PR!

Reply:
I think that was necessary with create-react-app, so once this lands we can and should move it out into a separate config file!